meshes/meshops.proto

```diff
@@ -7,11 +7,14 @@ package meshes;
 service MeshService {
     rpc CreateMeshInstance(CreateMeshInstanceRequest) returns (CreateMeshInstanceResponse) {}
     rpc MeshName(MeshNameRequest) returns (MeshNameResponse) {}
+    rpc MeshVersions(MeshVersionsRequest) returns (MeshVersionsResponse) {}
```

Comment:
Should we merge Mesh information to just one gRPC service? Even though we need to bring the breaking change. And I'd like to say that we need to follow the OOP message struct that would be more standard and easy to extend, right?

Reply:
MeshVersions is actually the versions of the service mesh available in the adapter, where as ComponentInfo gives the adapter version @Aisuko

Reply:
OK, I see and thanks.